OpenUpSA/wazimap-ng-ui
# showing spinner in location tag while loading parent geography

## Description
Showing a spinner while loading the data for the parent geography as we do when loading the data for a lower down geo

## Related Issue
https://github.com/OpenUpSA/wazimap-ng-ui/issues/385

## How to test it locally
* navigate to a geography
* click on a location tag(breadcrumb) of a parent geography
* confirm that a spinner is shown in the chip

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/121087723-7770f480-c7ed-11eb-9abe-c080c29373c7.png)


## Changelog

### Added

### Updated
* `src/js/elements/location_info_box.js` to send the click event parameter to use its target element along with the `location` object
* `src/js/setup/breadcrumbs.js` to make the spinner visible in the clicked element

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
